.github/workflows/behat.yml

```diff
@@ -0,0 +1,71 @@
+name: Behat
+on: [push, pull_request]
+jobs:
+    behat:
+        name: Behat
+        runs-on: ubuntu-latest
+        strategy:
+            matrix:
+                php: ['7.1', '7.2', '7.3', '7.4']
+        steps:
+            -   name: Setup PHP
+                uses: shivammathur/setup-php@v2
+                with:
+                    php-version: ${{ matrix.php }}
+                    extensions: mbstring, intl, gd, xml, dom, json, fileinfo, curl, zip, iconv
+            -   name: Setup MySQL
+                uses: mirromutth/mysql-action@v1.1
+                with:
+                    mysql version: '5.7'
+                    mysql database: 'prestashop'
+                    mysql root password: 'password'
+            -   name: Setup Node
+                uses: actions/setup-node@v2
+                with:
+                    node-version: '10'
+            -   uses: actions/checkout@v2
+                with:
+                    fetch-depth: 0
+
+            -   name: PrestaShop Configuration
+                run: cp .github/workflows/phpunit/parameters.yml app/config/parameters.yml
+
+            -   name: Get Composer Cache Directory
+                id: composer-cache
+                run: |
+                    echo "::set-output name=dir::$(composer config cache-files-dir)"
+
+            -   name: Cache Composer Directory
```

Comment:
the names are swapped :)

Reply:
Nice catch @mvorisek 
Can you provide a PR ? I can do it if you want !

Reply:
feel free to do so, I do not need any credit for it

Reply:
Done ! Thanks @mvorisek